console/console-init/ui/src/modules/device-detail/components/DeviceDetailHeader/DeviceDetailHeader.tsx

```diff
@@ -0,0 +1,192 @@
+import React, { useState } from "react";
+import {
+  Split,
+  SplitItem,
+  Title,
+  Flex,
+  FlexItem,
+  DropdownItem,
+  DropdownPosition,
+  PageSection,
+  PageSectionVariants,
+  Badge,
+  Switch,
+  DropdownSeparator
+} from "@patternfly/react-core";
+import { css, StyleSheet } from "@patternfly/react-styles";
+import { DropdownWithKebabToggle } from "components";
+import { FormatDistance } from "use-patternfly";
+
+interface IDeviceDetailHeaderProps {
+  deviceName: string;
+  addedDate: string;
+  lastTimeSeen: string;
+  isEnabled: boolean;
+  changeEnable: (value: boolean) => void;
+  onEdit: () => void;
+  onDelete: () => void;
+  onClone: () => void;
+}
+
+export const project_header_styles = StyleSheet.create({
+  flex_left_border: {
+    paddingLeft: "1em",
+    borderLeft: "0.05em solid",
+    borderRightColor: "lightgrey"
+  },
+  address_space_icon_margin: {
+    backgroundColor: "#EC7A08",
```

Comment:
address_space_icon_margin, address_icon_align are not being used and badge_style are not being used.